Source/BugsnagNotifier.m

```diff
@@ -335,10 +420,16 @@ - (void)orientationChanged:(NSNotification *)notif {
   case UIDeviceOrientationFaceDown:
     orientation = @"facedown";
     break;
-  case UIDeviceOrientationUnknown:
   default:
-    orientation = @"unknown";
+    return; // always ignore unknown breadcrumbs
+  }
+  
+  NSDictionary *lastBreadcrumb = [[self.configuration.breadcrumbs arrayValue] lastObject];
+    
+  if (lastBreadcrumb && [orientation isEqualToString:lastBreadcrumb[@"name"]]) {
```

Comment:
This check seems to not be working, though I think its because the last breadcrumb name isn't the correct field for a comparison. It looks like the name needs to indicate that this is an orientation breadcrumb and the orientation in `metadata` should be equal to `orientation` in the current scope.